BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -213,8 +213,9 @@ public class PortfolioStore: ObservableObject {
   @MainActor func fetchPriceHistory(
     for priceIds: [String]
   ) async -> [String: [BraveWallet.AssetTimePrice]] {
+    let uniquePriceIds = Array(Set(priceIds))
```

Comment:
Do we have to bridge it to Array? we can iterate over a `Set` too

Reply:
fixed in [edbc704](https://github.com/brave/brave-ios/pull/5838/commits/edbc704c503610a93ba78f438c39563a7f2805d0)